### ISSUE TITLE
Unable to use FM Radio

### ISSUE THREAD

**Issue by kylecorry31** [LineageOS/android_device_motorola_payton#4]
The FM radio seems to be missing some libraries on Lineage 16. FM radio apps from the Play Store are unable to detect the radio and the Motorola Radio application is incompatible.

**Comment by kylecorry31** [comment]
The latest update provides the FM radio application, but it will not play audio out of the speakers or headphones (but oddly enough, it allows audio to be recorded - and that recording is playable)

**Comment by erfanoabdi** [comment]
yeah moto FM audio paths is odd, you can use moto FM app to listen to it or other apps

**Comment by kylecorry31** [comment]
Unfortunately the Moto FM app won't download from play because it's incompatible for some reason and NextRadio can't detect the FM

**Comment by Moini** [comment]
This works now (for me, at least).

**Comment by kylecorry31** [comment]
I can't confirm, unfortunately - I switched to a different phone.
